### PR TITLE
Add graphql-hooks example

### PR DESCRIPTION
**Pull request by Joezo**

### CODE REVIEW

File: examples/with-graphql-hooks/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Deploy it to the cloud with [now](https://zeit.co/now) ([download](https://zeit.co/download)):
+
+```bash
+now
+```
+
+## The idea behind the example
+
+[GraphQL Hooks](https://github.com/nearform/graphql-hooks) is a library from NearForm that intends to be a minimal hooks-first GraphQL client. Providing it in a way familiar to Apollo users.
```
Comment (bmullan91) [inline examples/with-graphql-hooks/README.md]: Perhaps `Providing a similar API to Apollo`

File: examples/with-graphql-hooks/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+This started life as a copy of the `with-apollo` example. We then stripped out Apollo and replaced it with `graphql-hooks`. This was mostly as an exercise in ensuring basic functionality could be achieved in a similar way to Apollo.
```
Comment (bmullan91) [inline examples/with-graphql-hooks/README.md]: Maybe this should be at the top of the README? It be good document the bundle size difference between `with-apollo` and `with-graphql-hooks`

**Review by bmullan91 — APPROVED** [review]
Great work @Joezo  🙌

**Review by jackdclark — APPROVED** [review]
This is awesome, nice work @Joezo 👍

**Pull request closed without merging**